### PR TITLE
fix: site dark tweak style

### PR DESCRIPTION
**Pull request by ycjcl868**
### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Site / document update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |           |
| 🇨🇳 Chinese |           |

### ☑️ Self Check before Merge

- [ ] Doc is updated/provided or not needed
- [ ] Demo is updated/provided or not needed
- [ ] TypeScript definition is updated/provided or not needed
- [ ] Changelog is provided or not needed

-----
[View rendered components/button/index.en-US.md](https://github.com/ant-design/ant-design/blob/fix-dark-component/components/button/index.en-US.md)
[View rendered components/button/index.zh-CN.md](https://github.com/ant-design/ant-design/blob/fix-dark-component/components/button/index.zh-CN.md)
[View rendered components/table/demo/summary.md](https://github.com/ant-design/ant-design/blob/fix-dark-component/components/table/demo/summary.md)

### CODE REVIEW

File: components/button/style/mixin.less [hunk 1]
```diff
@@ -281,15 +281,33 @@
...
-    .button-color(
-      ~`colorPalette('@{error-color}', 5) `; @btn-default-bg; ~`colorPalette('@{error-color}', 5) `
-    );
+    & when (@theme = dark) {
+      .button-color(
+        ~`colorPalette('@{error-color}', 7) `; @btn-default-bg; ~`colorPalette('@{error-color}', 7)
+          `
+      );
+    }
+    & when not (@theme = dark) {
```
Comment (ycjcl868) [inline components/button/style/mixin.less]: 这块要改成主题变量的方式

File: site/theme/static/index.less [hunk 2]
```diff
@@ -1,4 +1,3 @@
-@import '../../../components/style/v2-compatible-reset.less';
```
Comment (afc163) [inline site/theme/static/index.less]: 删掉这个会有很多网站样式问题。

Comment (ycjcl868) [reply site/theme/static/index.less]: 好的，排查下

File: components/transfer/demo/table-transfer.md [hunk 3]
```diff
@@ -159,3 +159,9 @@ class App extends React.Component {
 
 ReactDOM.render(<App />, mountNode);
 ```
+
+<style>
+  [data-theme="dark"] .ant-transfer .ant-input[disabled] {
+    background-color: transparent;
```
Comment (afc163) [inline components/transfer/demo/table-transfer.md]: 这个应该放 dark 组件样式里。

Comment (ycjcl868) [reply components/transfer/demo/table-transfer.md]: ok

**Pull request merged**